src/features/onboarding/pages/OnboardingAuthentication.web.test.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+import { OnboardingAuthentication } from 'features/onboarding/pages/OnboardingAuthentication'
+import { render } from 'tests/utils'
+
+describe('OnboardingAuthentication', () => {
+  it('should render null in web', () => {
+    const component = render(<OnboardingAuthentication />)
+    expect(component).toMatchObject({})
+  })
```

Comment:
C'est pas mieux comme ça ? 

```suggestion
  it('should render null in web', () => {
    const renderAPI = render(<OnboardingAuthentication />)
    expect(renderAPI).toBeNull()
  })
```

Reply:
Ca marche qu'en native, je vais utiliser `toBeEmptyDOMElement` c'est mieux que `toMatchObject`